modules/react/common/spec/createCanvasTheme.spec.tsx

```diff
@@ -106,20 +129,6 @@ describe('createCanvasTheme', () => {
     expect(theme).toEqual(expected);
   });
 
-  test('custom theme should not override defaultCanvasTheme when merged', () => {
```

Comment:
The proxy doesn't do a merge so IMO there is no need for a cloneDeep or even this test